lang/python/python-mako/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-mako
+PKG_VERSION:=1.1.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=Mako-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/m/mako/
+PKG_HASH:=8195c8c1400ceb53496064314c6736719c6f25e7479cd24c77be3d9361cddc27
+
+HOST_BUILD_DEPENDS:=python3/host
+
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+PKG_LICENSE:=BSD
+PKG_LICENSE_FILES:=LICENSES.txt
+PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
```

Comment:
Is @commodo aware that you are adding him as maintainer?

Reply:
nope; i am not aware;
thanks for the heads-up;

i'd prefer not to take new packages from others;
it means i can't work on the ones that i am interested in;

however, i don't mind being added as co-maintainer;

Reply:
Next version of this patch will update the maintainer entry. Sorry for adding you, I copied the file from https://gist.github.com/lucize/b0691a79beff66a9ed2163e9f3999808 with you being mentioned as maintainer.